canonical-web-and-design/ubuntu.com
# 7425 appliance plex

## Done

- Created appliance/plex page

[copy](https://docs.google.com/document/d/1zgOl4GCK6O6Clv6oxp-DXDSKYIKO2pvpXaYXALiTlpo/edit) | [design](https://app.zeplin.io/project/5eb3d16a91c4117789940141/screen/5eb3fd0a59a84577c29a34ff) | [journey docs](https://docs.google.com/presentation/d/1v5O3iZ33FSrlsQwhESGp3XCvbfNqYLsLvymFxbQcIss/edit#slide=id.g84f83eaa6c_0_225)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/plex
- Change the select box and see that the download button URL changes (the pages don't exist yet)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #7425
